admin/products_to_categories.php

```diff
@@ -727,25 +727,25 @@ function toggle(source) {
                 </table>
                 <div class="form-group text-center">
                     <button type="submit" class="btn btn-primary floatButton"
-                            title="<?php echo BUTTON_UPDATE_CATEGORY_LINKS . " - " . $product_to_copy->fields['products_name']; ?>"><?php echo BUTTON_UPDATE_CATEGORY_LINKS . '<br><span>' . $product_to_copy->fields['products_model'] . '<br>' . $product_to_copy->fields['products_name'] . '<br>(#' . $products_filter . ')'; ?></span></button>
+                            title="<?= BUTTON_UPDATE_CATEGORY_LINKS . " - " . $product_to_copy->fields['products_name'] ?>"><?= BUTTON_UPDATE_CATEGORY_LINKS . '<br><span>' . $product_to_copy->fields['products_model'] . '<br>' . $product_to_copy->fields['products_name'] . '<br>(#' . $products_filter . ')' ?></span></button>
                 </div>
-                <?php echo '</form>'; ?>
+                <?= '</form>' ?>
             </div>
         </div>
     <?php } ?>
     <!-- Category Links eof -->
     <!-- Product-category links block eof-->
 
-    <div class="row"><?php echo zen_draw_separator('pixel_black.gif', '100%', '2'); ?></div>
+    <div class="row"><?= zen_draw_separator('pixel_black.gif', '100%', '2') ?></div>
 
     <!-- Global Tools -->
     <div class="col-lg-12">
-        <h2><?php echo HEADER_CATEGORIES_GLOBAL_TOOLS; ?></h2>
+        <h2><?= HEADER_CATEGORIES_GLOBAL_TOOLS ?></h2>
         <!-- Copy linked categories from one product to another -->
         <div class="row dataTableHeadingRow">
-            <h3><?php echo TEXT_HEADING_COPY_LINKED_CATEGORIES; ?></h3>
+            <h3><?= TEXT_HEADING_COPY_LINKED_CATEGORIES ?></h3>
             <div class="form-group-row">
-                <?php echo sprintf(TEXT_INFO_COPY_LINKED_CATEGORIES, ($products_filter > 0 ? ': <strong>' . $source_product_details . '</strong><br>' : ' ')); ?>
+                <?= sprintf(TEXT_INFO_COPY_LINKED_CATEGORIES, ($products_filter > 0 ? ':<p><strong>' . $source_product_details . '</strong></p>' : ' ')); ?>
```

Comment:
I presume the addition of the `<p>` tag was to create more padding around the text, vs merely the `<br>` tag?

Reply:
Yes and on it's own line.